realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -305,6 +306,41 @@ public void removeChangeListener(OrderedRealmCollectionChangeListener<RealmResul
     }
 
     /**
+     * Returns an Rx Observable that monitors changes to this RealmResults. It will emit the current RealmResults when
+     * subscribed to. For each update to the RealmResult a pair consisting of the RealmResults and the
+     * {@link OrderedCollectionChangeSet} will be sent. The changeset will be {@code null} the first
+     * time an RealmResults is emitted.
+     * <p>
+     * Changeset observables do not support backpressure as each {@link OrderedCollectionChangeSet} depends on
```

Comment:
Is this comment necessary?  Isn't that the distinction between `Flowable` and `Observable`?